### PR TITLE
Use the struct and constructor like the other elements

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: contribs/gmf/externs/gmf-themes.js [hunk 1]
```diff
@@ -89,7 +105,7 @@ gmfThemes.GmfTheme.prototype.functionalities;
  * neither a WMS group.
  * This represent « first level group » (Block in the layer tree),
  * or all sub nodes that's not al leaf.
- * @constructor
+ * @record
  * @struct
  * @extends gmfThemes.GmfBaseNode
```
Comment (gberaudo) [inline contribs/gmf/externs/gmf-themes.js]: Do `@record` works together with `@extends`?

File: externs/mapfish-print-v3.js [hunk 2]
```diff
@@ -366,9 +383,57 @@ var MapFishPrintSymbolizer = function() {};
 MapFishPrintSymbolizer.prototype.type;
 
 
+/**
+ * @type {string}
+ */
+MapFishPrintSymbolizer.prototype.fillColor;
```
Comment (gberaudo) [inline externs/mapfish-print-v3.js]: Should add `|undefined` for optional attributes.

File: externs/mapfish-print-v3.js [hunk 3]
```diff
@@ -530,7 +597,8 @@ MapFishPrintSymbolizerPolygon.prototype.strokeDashstyle;
 
 
 /**
- * @constructor
+ * @record
+  @struct
```
Comment (gberaudo) [inline externs/mapfish-print-v3.js]: Missing a star.

File: contribs/gmf/externs/gmf-themes.js (hunk 1 above)
Comment (sbrunner) [inline contribs/gmf/externs/gmf-themes.js]: It seem yes :-)

**Pull request merged**